include/odp/api/spec/packet.h

```diff
@@ -155,6 +155,25 @@ int odp_packet_reset(odp_packet_t pkt, uint32_t len);
  * @return Maximum packet data length for odp_packet_reset()
  */
 uint32_t odp_packet_reset_max_len(odp_packet_t pkt);
+
+/**
+ * Reset packet metadata
+ *
+ * Resets all packet metadata, except the ones listed below, to their default
+ * values:
```

Comment:
Suggesting a simplified sentence:

"Resets all packet metadata to their default values, except the ones listed below:"

Reply:
Changed in V2.